src/core/rtkQuery/splitV3Api/collectionApi.ts

```diff
@@ -47,8 +50,21 @@ const collectionApi = splitV3Api.injectEndpoints({
         params: { includeEmpty, topLevelOnly },
       }),
     }),
-    getGroupSeries: build.query<SeriesType[], { groupId?: string }>({
-      query: ({ groupId }) => ({ url: `Group/${groupId}/Series` }),
+    getGroupSeries: build.query<InfiniteResultType<SeriesType[]>, { groupId?: string, randomImages?: boolean }>({
```

Comment:
Should probably be `groupId: string`, not `?: string`

Reply:
done